<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertRegistrationForm</class>
 <widget class="QDialog" name="insertRegistrationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nameField"/>
    </item>
    <item>
     <widget class="QLineEdit" name="actionField"/>
    </item>
    <item>
     <widget class="QComboBox" name="methodField">
      <item>
       <property name="text">
        <string>GET</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>POST</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Form Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Form Action:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Method:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>32</width>
     <height>107</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QToolButton" name="deleteButton">
      <property name="text">
       <string>Delete Selected Item</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="moveUpButton">
      <property name="text">
       <string>Move Up</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons/go-up.png</normaloff>:/icons/go-up.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="moveDownButton">
      <property name="text">
       <string>Move Down</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons/go-down.png</normaloff>:/icons/go-down.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>411</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="bigList"/>
    </item>
    <item>
     <widget class="QToolButton" name="addButton">
      <property name="text">
       <string>Add To List</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons/arrow-right.png</normaloff>:/icons/arrow-right.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="smallList"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>insertRegistrationForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>insertRegistrationForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
